source/accessible-media/production-audio-description.html.erb.md

```diff
@@ -135,58 +128,51 @@ screen readers can read aloud as the text is delivered. See [examples of
 TTS descriptions](http://ncamftp.wgbh.org/ibm/dvs/). The basic workflow
 for TTS audio descriptions generally follows this pattern:
 
-### Basic workflow for creating pre-produced TTS audio descriptions:
-
-{::nomarkdown}
-<%= sample_start %>
-{:/nomarkdown}
-
 1. **Watch the video**<br>
     Note opportunities for descriptive narration (i.e., pauses in
     narration or dialog into which descriptions can be inserted).
+
 2. **Write the description script**<br>
-    Using a **caption-editing tool**, write the description script that
```

Comment:
Remove highlighting. It is a bit weird to use a caption-editing tool, although I understand that they might have useful functions.